src/main/java/games/strategy/triplea/ai/proAI/ProTechAI.java

```diff
@@ -40,8 +40,9 @@ static void tech(final ITechDelegate techDelegate, final GameData data, final Pl
       return;
     }
     final Territory myCapitol = TerritoryAttachment.getFirstOwnedCapitalOrFirstUnownedCapital(player, data);
-    final float enemyStrength = getStrengthOfPotentialAttackers(myCapitol, data, player, false, true, null);
-    float myStrength = strength(myCapitol.getUnits().getUnits(), false, false, false);
+    final float enemyStrength = getStrengthOfPotentialAttackers(myCapitol, data, player);
+    float myStrength = (myCapitol == null || myCapitol.getUnits() == null) ? 0.0F :
+            strength(myCapitol.getUnits().getUnits(), false, false, false);
```

Comment:
@DanVanAtta Can you check your formatter settings on your new rig?  Your IDE is indenting wrapped lines with 8 spaces instead of the 4 spaces our formatter configuration specifies.  Just trying to avoid ping-ponging between Eclipse and IntelliJ when any of us run the formatter.

Reply:
Formatting is off, yeah, we need to check-in/share the settings and document it... It's not a matter of ping/pong, we checked in the config for one IDE but not the other. https://github.com/triplea-game/triplea/issues/1794 is a reminder to get that set up.

Reply:
Formatting fixed in: fe2979e, out of band checked in the intellij config and documented the plugins needed. I hope that addresses this so we can start benefiting from the automation once again :+1: